src/main/resources/com/google/api/codegen/java/settings.snip

```diff
@@ -142,13 +163,13 @@
 
   @if xsettingsClass.hasDefaultServiceAddress
     /** Returns a builder for the default ChannelProvider for this service. */
-    public static GrpcTransportProvider.Builder defaultGrpcTransportProviderBuilder() {
-      return GrpcTransportProvider.newBuilder()
-          .setChannelProvider(defaultGrpcChannelProviderBuilder().build());
+    public static {@transportProtocolName}TransportProvider.Builder default{@transportProtocolName}TransportProviderBuilder() {
```

Comment:
Don't compose tokens in the template, compose them in the transformer and put them in the view model

Reply:
Done.